src/main/resources/application.yaml

```diff
@@ -101,7 +104,8 @@ spring:
     mode: TEXT
     suffix: .sql
 
-  jersey.application-path: rest
+  # this must be `camunda` to match module descriptor path patterns
+  jersey.application-path: camunda
```

Comment:
This must match module descriptor and probably should be a template variable. For now, hardcoding it.

Reply:
If and when changed, mod-workflow property `okapi.workflow-engine.path` must be updated accordingly. 

https://github.com/folio-org/mod-workflow/pull/116/files#diff-5f1399fae635ac57e696d503eb2135f7575b3a59a486e21613d3343f8f2a3243R50